### PR TITLE
283 Add title heading element to display h1 content in page tab title

### PR DESCRIPTION
**Pull request by riatzukiza**
### This pull request changes...

- Title matches h1 contents on edit profile page

### [AC] This pull request is ready to merge when...

- [x] Tests have been updated (and all tests are passing)
- [ ] Vulnerability Scan Test (Zap) has no high/critcial CVEs
- [x] Linting Tests (ESLint for UI and Flake8 for Python)
- [x] Unit Tests ( including a code coverage test report)
- [ ] Accessibility Tests (Pa11y UI Only)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate

### [DoD] This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: tdrs-frontend/src/components/EditProfile/EditProfile.jsx [hunk 1]
```diff
@@ -106,128 +118,87 @@ function EditProfile() {
...
     setTouched(formValidation.touched)
-    setTimeout(() => errorRef.current.focus(), 0)
+
+    if (!Object.values(formValidation.errors).length) {
+      return dispatch(requestAccess(profileInfo))
+    }
+    return setTimeout(() => errorRef.current.focus(), 10)
+  }
+
+  if (requestedAccess) {
```
Comment (riatzukiza) [inline tdrs-frontend/src/components/EditProfile/EditProfile.jsx:128]: There was a line here causing a test to fail `setTimeout(() => errorRef.current.focus(), 10)`. Removing it caused the tests to pass, I want to be sure this is correct.

Comment (hilvitzs) [reply tdrs-frontend/src/components/EditProfile/EditProfile.jsx:128]: Sorry, I'm a little confused. I just pulled it down and that line `setTimeout(() => errorRef.current.focus(), 10)` is present and the tests are passing. That focus is necessary for an accessibility concern that Miles found.

Comment (hilvitzs) [reply tdrs-frontend/src/components/EditProfile/EditProfile.jsx:128]: On the 231 branch, this was changed to a timeout of 0. I think just making sure that this branch is current with 231 should have everything working correctly.

File: tdrs-frontend/src/components/EditProfile/EditProfile.jsx [hunk 2]
```diff
@@ -49,6 +49,7 @@ export const validation = (fieldName, fieldValue) => {
  * There is an input for first and last name and a combo box to select
```
Comment (carltonsmith) [inline tdrs-frontend/src/components/EditProfile/EditProfile.jsx:49]: @riatzukiza was the work in this file done for this branch, or is there another branch we have pulled in here?

File: tdrs-frontend/src/components/EditProfile/EditProfile.test.js [hunk 3]
```diff
@@ -19,18 +19,31 @@ describe('EditProfile', () => {
...
-      <Provider store={store}>
-        <EditProfile />
-      </Provider>
-    )
-
-    const h1 = wrapper.find('h1')
-    expect(h1).toExist()
-    expect(h1.text()).toEqual('Request Access')
-  })
+  // testing in private template now
```
Comment (carltonsmith) [inline tdrs-frontend/src/components/EditProfile/EditProfile.test.js:22]: Should these tests be commented out?

**Review by carltonsmith — COMMENTED** [review]
Looks great for the most part, just added a couple of questions

**Review by carltonsmith — CHANGES_REQUESTED** [review]
The edit profile page isn't displaying correctly when I run this locally:
Screen Shot 2020-11-17 at 1 50 15 PM

File: tdrs-frontend/src/components/EditProfile/EditProfile.jsx (hunk 2 above)
Comment (riatzukiza) [reply tdrs-frontend/src/components/EditProfile/EditProfile.jsx:49]: This is all from this branch

File: tdrs-frontend/src/components/EditProfile/EditProfile.test.js (hunk 3 above)
Comment (riatzukiza) [reply tdrs-frontend/src/components/EditProfile/EditProfile.test.js:22]: I'll remove them

Comment (riatzukiza) [reply tdrs-frontend/src/components/EditProfile/EditProfile.test.js]: and their gone.

**Review by carltonsmith — APPROVED** [review]

**Comment by carltonsmith** [comment]
@alexsoble This hasn't deployed yet as 197 is under review. Once that has been reviewed we'll deploy it so you can review it operationally. In the meantime if you can informally review this we'd appreciate it.

**Pull request merged**